pipelines/binary-builder.yml

```diff
@@ -101,41 +97,41 @@ resources: #####################################################################
   - name: docker-cflinuxfs2-rootfs
     type: docker-image
     source:
-      repository: cloudfoundry/cflinuxfs2
+      repository: jandubois/opensusefs2
       username: {{buildpacks-docker-username}}
       password: {{buildpacks-docker-password}}
       email: cf-buildpacks-eng@pivotal.io
 
-<% dep_build_names = %w(bower go bundler composer dotnet glide godep httpd hwc jruby php php7 python node nginx ruby yarn) %>
+<% dep_build_names = %w(bower go bundler composer glide godep httpd jruby php php7 python node nginx ruby yarn) %>
 <% dep_build_names.each do |dep| %>
   - name: buildpacks-bucket-<%= dep %>
     type: s3
     source:
       bucket: {{buildpacks-binaries-s3-bucket}}
       access_key_id: {{pivotal-buildpacks-s3-access-key}}
       secret_access_key: {{pivotal-buildpacks-s3-secret-key}}
-      regexp: dependencies/<%= dep %>/<%= dep %>.*(\d+.\d+.\d+)(.*)
+      regexp: dependencies/<%= dep %>/(.*)
+      endpoint: {{buildpacks-binaries-s3-endpoint}}
 <% end %>
 
 groups: ###############################################################################################################
 
   - name: enqueue-automated-builds
     jobs:
-<% %w(bower composer dotnet glide godep hwc nginx node yarn).each do |auto_dep| %>
+<% %w(bower composer glide godep nginx node yarn).each do |auto_dep| %>
```

Comment:
I'm getting the feeling this list should have been refactored to somewhere else at the top of this file